core/src/com/unciv/logic/civilization/QuestManager.kt

```diff
@@ -358,14 +397,29 @@ class QuestManager {
 
     /** Returns the score for the [assignedQuest] */
     private fun getScoreForQuest(assignedQuest: AssignedQuest): Int {
-        @Suppress("UNUSED_VARIABLE")  // This is a work in progress
         val assignee = civInfo.gameInfo.getCivilization(assignedQuest.assignee)
         return when (assignedQuest.questName) {
-            // Waiting for contest quests
+            QuestName.ContestCulture.value -> assignee.policies.totalCulture - assignedQuest.data1.toInt()
+            QuestName.ContestFaith.value -> assignee.religionManager.totalFaith - assignedQuest.data1.toInt()
+            QuestName.ContestTech.value -> assignee.tech.getNumberOfTechsResearched() - assignedQuest.data1.toInt()
```

Comment:
On first glance this looks like researching 'future tech' doesn't count towards this, is this correct and intentional?

Reply:
Yes, correct and intentional.